guides/common/modules/proc_creating-a-custom-certificate.adoc

```diff
@@ -0,0 +1,85 @@
+[id="creating-a-custom-certificate_{context}"]
+
+= Creating a Custom SSL Certificate for {ProductName}
```

Comment:
I understand that the ProductName is set by the context to either ProjectServer or SmartProxyServer.
However, can we please rename this variable to make it clearer? E.g. `{ConditionalServer}`?
Unless it's used consistently with other modules that would require the same change. That would be out of scope.

Reply:
I checked and this is used with other modules. @maximiliankolb any thoughts on this?

Reply:
Yes, having a descriptive attribute would be nice. It's something for another PR.